using_the_api.md

```diff
@@ -179,15 +192,36 @@ The structure of the post request allows for a flexible and complex query with b
       "value": "treasury"
     }
     ```
-    * `fy` - Evaluates if the field (generally should be a datetime field) falls within the federal fiscal year specified as `value`. `value` should be a 4-digit integer specifying the fiscal year. An example of a fiscal year is FY 2017 which runs from October 1st 2016 to September 30th 2017.
+    * `fy` - Evaluates if the field (generally should be a datetime field) falls within the federal fiscal year specified as `value`. `value` should be a 4-digit integer specifying the fiscal year. An example of a fiscal year is FY 2017 which runs from October 1st 2016 to September 30th 2017. Does not need `value_format` as it is assumed.
     ```
     {
       "field": "date_signed",
       "operation": "fy",
       "value": 2017
     }
     ```
+    * `range_intersect` - Evaluates if the range defined by a two-field list intersects with the range defined
```

Comment:
Can you be more explicit about the use case? I.e. finding overlap between the fiscal year range and the period of performance range? I think it will help people grok it.